src/app/components/schedule-card/ScheduleCard.jsx

```diff
@@ -2,20 +2,18 @@ import "./ScheduleCard.scss";
 
 import { scheduleData } from "../../data/schedule-info";
 
-const Event = ({ name, description, time, tooltip }) => (
+const Event = ({ name, description, time, tooltip}) => (
```

Comment:
minor: space between `tooltip` and the right bracket (not sure if we have a lint script to help auto format but i should mark that down as a todo for next year)